### PR TITLE
Fix a bunch of compiler warnings

### PR DESCRIPTION
**Pull request by Visne**
190 of them, to be exact

I was too scared to touch obsoletion warnings for now, but I might look at those later.

### CODE REVIEW

**Comment by vulppine** [comment]
👍

File: Content.Client/Atmos/Monitor/AtmosMonitorVisualizer.cs [hunk 1]
```diff
@@ -26,13 +26,15 @@ public sealed class AtmosMonitorVisualizer : AppearanceVisualizer
         [DataField("setOnDepowered")]
         private readonly Dictionary<string, string>? _setOnDepowered;
 
+        [Obsolete("Subscribe to your component being initialised instead.")]
```
Comment (ShadowCommander) [inline Content.Client/Atmos/Monitor/AtmosMonitorVisualizer.cs:29]: Are these not going to be changed? If they are, is there another place that says to change them like an issue?

Comment (vulppine) [reply Content.Client/Atmos/Monitor/AtmosMonitorVisualizer.cs:29]: this is a good point: i think these should at least have a link to the instructions for the AppearanceVisualizer to VisualizerSystem<T> port: https://docs.spacestation14.io/en/engine/porting-visualizers

Comment (Visne) [reply Content.Client/Atmos/Monitor/AtmosMonitorVisualizer.cs:29]: The method they are overriding are obsolete, therefore these should be obsolete as well. It gives a compiler warning, which is why I am adding Obsolete attributes here.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**